frontend/scripts/actions/tool.actions.js

```diff
@@ -420,7 +420,7 @@ export function loadLinks() {
     }
 
     const selectedBiomeFilter = state.tool.selectedBiomeFilter;
-    if (selectedBiomeFilter && selectedBiomeFilter.name && selectedBiomeFilter.value !== 'none') {
+    if (selectedBiomeFilter && selectedBiomeFilter.name && selectedBiomeFilter.name !== 'none') {
```

Comment:
did you test this in the browser to make sure it doesn't break something?

Reply:
yep